jetty-websocket/websocket-javax-common/src/main/java/org/eclipse/jetty/websocket/javax/common/JavaxMessagePartialMethodHandle.java

```diff
@@ -0,0 +1,84 @@
+//
+// ========================================================================
+// Copyright (c) 1995-2022 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.websocket.javax.common;
+
+import java.lang.invoke.MethodHandle;
+import java.lang.invoke.WrongMethodTypeException;
+import javax.websocket.MessageHandler;
+
+import org.eclipse.jetty.websocket.core.internal.util.JettyMethodHandle;
+
+class JavaxMessagePartialMethodHandle<T> implements JettyMethodHandle
```

Comment:
Again the naming would be better if it was:
```suggestion
class JavaxMessagePartialMethodHolder<T> implements MethodHolder
```